### PR TITLE
Revert "Merge pull request #2482 from rina23q/improve/1718/get-operat…

### PR DESCRIPTION
**Pull request by rina23q**
…ion-from-json-over-mqtt"

This reverts commit 9e204819055dabe4841cd19528809b7fc5336e9a, reversing changes made to 71ad70a0b4a4b7411c4352e24b78400feb757bc4.

## Proposed changes
This PR reverts the change introduced by #2482. It has to subscribe c8y's devicecontrol topic and needed to add it to bridge file. The issue is captured https://github.com/thin-edge/thin-edge.io/issues/2545

Reverted by 
```
git revert -m 1 9e204819055dabe4841cd19528809b7fc5336e9a
```

Please check if your change after 9e204819055dabe4841cd19528809b7fc5336e9a is still there.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
https://github.com/thin-edge/thin-edge.io/issues/2545

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: crates/core/tedge/src/cli/connect/bridge_config_c8y.rs [hunk 1]
```diff
@@ -57,7 +57,6 @@ impl From<BridgeConfigC8yParams> for BridgeConfig {
             r#"measurement/measurements/create out 2 c8y/ """#.into(),
             r#"event/events/create out 2 c8y/ """#.into(),
             r#"alarm/alarms/create out 2 c8y/ """#.into(),
-            r#"devicecontrol/notifications in 2 c8y/ """#.into(),
```
Comment (didier-wenzek) [inline crates/core/tedge/src/cli/connect/bridge_config_c8y.rs:60]: This change can safely be included in the release without being reverted.

File: crates/core/tedge/src/cli/connect/bridge_config_c8y.rs [hunk 2]
```diff
@@ -198,7 +197,6 @@ fn test_bridge_config_from_c8y_params() -> anyhow::Result<()> {
...
```
Comment (didier-wenzek) [inline crates/core/tedge/src/cli/connect/bridge_config_c8y.rs:201]: This change can safely be included in the release without being reverted.

**Review by albinsuresh — APPROVED** [review]
LGTM

**Review by didier-wenzek — COMMENTED** [review]
I confirm the revert is not affecting the PRs related to operation workflows.

File: crates/core/tedge/src/cli/connect/bridge_config_c8y.rs (hunk 1 above)
Comment (rina23q) [reply crates/core/tedge/src/cli/connect/bridge_config_c8y.rs:60]: I'll create a follow-up PR to include some stuff. Also, the bug fix (duplicated "c8y" on cmd ID) is going to be reverted.

**Pull request merged**